<p id="agp">
  <strong>AGP</strong> (.agp) - the Accessioned Golden Path (AGP) file describes
  the assembly of a larger sequence object from smaller objects:
</p>
<pre>
chr1 1       1972671 0 W scaffold43  1 1972671 m
chr1 1972672 3061819 1 W scaffold8   1 1089148 p
chr1 3061820 3181505 2 W scaffold548 1 119686  m
chr1 3181506 4176151 3 W scaffold313 1 994646  m</pre
>
<p>
  The large object can be a contig, a scaffold (supercontig), or a chromosome.
  See
  <a
    href="http://www.ncbi.nlm.nih.gov/projects/genome/assembly/agp/AGP_Specification.shtml"
    target="_blank"
    >AGP Specification v2.0</a
  >
</p>
<p id="bam">
  <strong>BAM</strong> (.bam) - the
  <a href="http://genome.sph.umich.edu/wiki/BAM" target="_blank"
    >Binary Alignment/Map (BAM) format</a
  >
  is the compressed binary version of the Sequence Alignment/Map (SAM) format, a
  compact and index-able representation of nucleotide sequence alignments.
</p>
<p id="bigwig">
  <strong>BIGWIG</strong> (.bw) - the
  <a href="http://genome.ucsc.edu/goldenPath/help/bigWig.html" target="_blank"
    >BIGWIG format</a
  >
  is for storing dense, continuous data (such as GC percent, probability scores,
  and transcriptome data) that will be displayed in the UCSC Genome Browser as a
  graph. BIGWIG files are created initially from wiggle (WIG) type files, using
  the program wigToBigWig.
</p>
<p id="chain">
  <strong>CHAIN</strong> (.chain) - the
  <a href="http://genome.ucsc.edu/goldenPath/help/chain.html" target="_blank"
    >CHAIN format</a
  >
  describes a pairwise alignment that allow gaps in both sequences
  simultaneously and is used by the UCSC Genome Browser.
</p>
<p id="contig">
  <strong>CONTIG</strong> (.contig) - the
  <a href="http://soap.genomics.org.cn/soapdenovo.html" target="_blank"
    >CONTIG format</a
  >
  is a direct output from the SOAPdenovo alignment program:
</p>
<pre>
>1 length 32 cvg_0.0_tip_0
GAGAACGGCGAAGCCTGCTCGGGCCCGTTATA
>3 length 32 cvg_23.0_tip_0
TAGCAGCGATTTGATCAAACTCAATCTTACCG
>5 length 32 cvg_40.0_tip_0
GGTAAGATTGAGTTTGATCAAATCGCTGCTAT</pre
>
<p id="excel">
  <strong>EXCEL</strong> (.xls, .xlsx) - Microsoft office spreadsheet files
</p>
<p id="fasta">
  <strong>FASTA</strong> (.fasta, .fa, .seq, .cds, .pep, .scafseq [SOAPdenovo
  output file -- sequence of each scaffold]) -
  <a href="http://www.ebi.ac.uk/help/formats.html#fasta" target="_blank"
    >FASTA</a
  >
  is a text-based format for representing either nucleotide sequences or peptide
  sequences.
</p>
<p id="fastq">
  <strong>FASTQ</strong> (.fq, .fastq) - the
  <a href="http://maq.sourceforge.net/fastq.shtml" target="_blank"
    >FASTQ format</a
  >
  stores sequences (usually nucleotide sequence) and Phred qualities in a single
  file.
</p>
<p id="gff">
  <strong>GFF</strong> (.gff) - The
  <a href="http://www.sanger.ac.uk/resources/software/gff/" target="_blank"
    >General Feature Format (GFF)</a
  >
  is used for describing genes and other features of DNA, RNA and protein
  sequences.
</p>
<p id="ipr">
  <strong>IPR</strong> (.ipr) - the
  <a
    href="http://wego.genomics.org.cn/cgi-bin/wego/Documents.pl"
    target="_blank"
    >Web Gene Ontology (WEGO) Annotation format</a
  >
  consists of the protein ID, followed by column(s) that are the IPR (<a
    href="http://www.ebi.ac.uk/interpro/"
    target="_blank"
    >InterPro</a
  >) ID(s):
</p>
<pre>
CR_ENSP00000334840
CR_ENSMMUP00000018123 IPR000504 IPR003954
CR_ENSP00000333725    IPR001781 IPR015880 IPR007087 IPR001909</pre
>
<p>
  See
  <a
    href="http://www.ncbi.nlm.nih.gov/pubmed/16845012?dopt=Abstract"
    target="_blank"
    >WEGO: a web tool for plotting GO annotations</a
  >
</p>
<p id="kegg">
  <strong>KEGG</strong> (.kegg) - the
  <a
    href="http://wego.genomics.org.cn/cgi-bin/wego/Documents.pl"
    target="_blank"
    >Web Gene Ontology (WEGO) Annotation format</a
  >
  consists of the protein ID, followed by column(s) that are the
  <a href="http://www.genome.jp/kegg/" target="_blank">KEGG</a> (Kyoto
  Encyclopedia of Genes and Genomes) ID(s):
</p>
<pre>
CR_ENSMMUP00000031408 ko03010
CR_ENSP00000364815    ko00970 ko00290
CR_ENSP00000414605    ko05146 ko04510 ko04512</pre
>
<p>
  See
  <a
    href="http://www.ncbi.nlm.nih.gov/pubmed/16845012?dopt=Abstract"
    target="_blank"
    >WEGO: a web tool for plotting GO annotations</a
  >
</p>
<p id="maf">
  <strong>MAF</strong> (.maf) - the
  <a href="http://genome.ucsc.edu/FAQ/FAQformat.html#format5" target="_blank"
    >Multiple Alignment Format (MAF)</a
  >
  stores a series of multiple alignments at the DNA level between entire
  genomes.
</p>
<p id="net">
  <strong>NET</strong> (.net) - the
  <a href="http://genome.ucsc.edu/goldenPath/help/net.html" target="_blank"
    >NET file format</a
  >
  is used to describe the axtNet data that underlie the net alignment
  annotations in the UCSC Genome Browser.
</p>
<p id="pdf"><strong>PDF</strong> (.pdf) - portable document format</p>
<p id="png"><strong>PNG</strong> (.png) - portable network graphics</p>
<p id="qmap">
  <strong>QMAP</strong> (.qmap) -
  <a
    href="http://yh.genomics.org.cn/Format%20of%20qmap%20file.txt"
    target="_blank"
    >QMAP</a
  >
  files are generated for methylation data from an internal
  <a href="http://www.genomics.cn/en" target="_blank">BGI</a> pipeline.
</p>
<p id="qual">
  <strong>QUAL</strong> (.qual) - the
  <a
    href="http://www.broadinstitute.org/crd/wiki/index.php/Qual"
    target="_blank"
    >QUAL file format</a
  >
  represents base quality score file for NextGen data (similar in format to
  fasta).
</p>
<p id="rpkm">
  <strong>RPKM</strong> (.rpkm) - Gene expression levels are calculated by
  <a
    href="http://sourceforge.net/apps/mediawiki/seqgene/index.php?title=SeqGene#.rpkm:_expression_estimates_generated_by_rpkm.py"
    target="_blank"
    >Reads Per Kilobase per Million (RPKM) mapped reads</a
  >
  eg 1kb transcript with 1000 alignments in a sample of 10 million reads (out of
  which 8 million reads can be mapped) will have RPKM = 1000/(1 * 8) = 125:
</p>
<pre>
ENSP00000379387 15.5651433366423 6002951 289 3093
ENSP00000349977 24.7483107230444 6002951 398 2679
ENSP00000368887 24.6477413647837 6002951 174 1176</pre
>
<p id="sam">
  <strong>SAM</strong> (.sam) - the
  <a href="http://genome.sph.umich.edu/wiki/SAM" target="_blank"
    >Sequence Alignment/Map (SAM) format</a
  >
  is a TAB-delimited text format consisting of a header section, which is
  optional, and an alignment section. Most often it is generated as a human
  readable version of its sister
  <a href="http://gigadb.org/site/help#bam" target="_blank">BAM</a> format,
  which stores the same data in a compressed, indexed, binary form. Currently,
  most SAM format data is output from aligners that read
  <a href="http://gigadb.org/site/help#fastq" target="_blank">FASTQ</a> files
  and assign the sequences to a position with respect to a known reference
  genome. In the future, SAM will also be used to archive unaligned sequence
  data generated directly from sequencing machines. See
  <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002/" target="_blank"
    >The Sequence Alignment/Map format and SAMtools</a
  >
</p>
<p id="tar"><strong>TAR</strong> (.tar) - an archive containing other files</p>
<p id="text">
  <strong>TEXT</strong> (.doc, .readme, .text, .txt) - a text file
</p>
<p id="vcf">
  <strong>VCF</strong> (.vcf) - the
  <a
    href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-40"
    target="_blank"
    >Variant Call Format (VCF)</a
  >
  is a text file format for representing eg SNPs, InDels, CNVs, SVs,
  microsatellites, genotypes.
</p>
<p id="wego">
  <strong>WEGO</strong> (.wego) - the
  <a
    href="http://wego.genomics.org.cn/cgi-bin/wego/Documents.pl"
    target="_blank"
    >Web Gene Ontology (WEGO) Annotation format</a
  >
  consists of the protein ID, followed by column(s) that are the
  <a href="http://www.geneontology.org/" target="_blank">GO</a> ID(s):
</p>
<pre>
Bmb015379_2_IPR001092
Bmb003749_1_IPR006329 GO:0009168 GO:0003876
Bmb006173_1_IPR000909 GO:0007165 GO:0004629 GO:0007242<</pre
>
<p>
  See
  <a
    href="http://www.ncbi.nlm.nih.gov/pubmed/16845012?dopt=Abstract"
    target="_blank"
    >WEGO: a web tool for plotting GO annotations</a
  >
</p>
<p id="wig">
  <strong>WIG</strong> (.wig) - the output file from
  <a href="http://tophat.cbcb.umd.edu/" target="_blank">TopHat</a> is a
  <a href="http://genome.ucsc.edu/goldenPath/help/wiggle.html" target="_blank"
    >UCSC wigglegram</a
  >
  of alignment coverage.
</p>
<p id="unknown"><strong>UNKNOWN</strong> - any file format not in this list</p>
<p id="xml">
  <strong>XML</strong> (.xml) -
  <a href="http://www.w3schools.com/xml/" target="_blank"
    >eXtensible Markup Language</a
  >
</p>
