AGP (.agp) - the Accessioned Golden Path (AGP) file describes the assembly of a larger sequence object from smaller objects: chr1 1 1972671 0 W scaffold43 1 1972671 m chr1 1972672 3061819 1 W scaffold8 1 1089148 p chr1 3061820 3181505 2 W scaffold548 1 119686 m chr1 3181506 4176151 3 W scaffold313 1 994646 m The large object can be a contig, a scaffold (supercontig), or a chromosome. See AGP Specification v2.0 BAM (.bam) - the Binary Alignment/Map (BAM) format is the compressed binary version of the Sequence Alignment/Map (SAM) format, a compact and index-able representation of nucleotide sequence alignments. BIGWIG (.bw) - the BIGWIG format is for storing dense, continuous data (such as GC percent, probability scores, and transcriptome data) that will be displayed in the UCSC Genome Browser as a graph. BIGWIG files are created initially from wiggle (WIG) type files, using the program wigToBigWig. CHAIN (.chain) - the CHAIN format describes a pairwise alignment that allow gaps in both sequences simultaneously and is used by the UCSC Genome Browser. CONTIG (.contig) - the CONTIG format is a direct output from the SOAPdenovo alignment program: >1 length 32 cvg_0.0_tip_0 GAGAACGGCGAAGCCTGCTCGGGCCCGTTATA >3 length 32 cvg_23.0_tip_0 TAGCAGCGATTTGATCAAACTCAATCTTACCG >5 length 32 cvg_40.0_tip_0 GGTAAGATTGAGTTTGATCAAATCGCTGCTAT EXCEL (.xls, .xlsx) - Microsoft office spreadsheet files FASTA (.fasta, .fa, .seq, .cds, .pep, .scafseq [SOAPdenovo output file - sequence of each scaffold]) - FASTA is a text-based format for representing either nucleotide sequences or peptide sequences. FASTQ (.fq, .fastq) - the FASTQ format stores sequences (usually nucleotide sequence) and Phred qualities in a single file. GFF (.gff) - The General Feature Format (GFF) is used for describing genes and other features of DNA, RNA and protein sequences. IPR (.ipr) - the Web Gene Ontology (WEGO) Annotation format consists of the protein ID, followed by column(s) that are the IPR (InterPro) ID(s): CR_ENSP00000334840 CR_ENSMMUP00000018123 IPR000504 IPR003954 CR_ENSP00000333725 IPR001781 IPR015880 IPR007087 IPR001909 See WEGO: a web tool for plotting GO annotations KEGG (.kegg) - the Web Gene Ontology (WEGO) Annotation format consists of the protein ID, followed by column(s) that are the KEGG (Kyoto Encyclopedia of Genes and Genomes) ID(s): CR_ENSMMUP00000031408 ko03010 CR_ENSP00000364815 ko00970 ko00290 CR_ENSP00000414605 ko05146 ko04510 ko04512 See WEGO: a web tool for plotting GO annotations MAF (.maf) - the Multiple Alignment Format (MAF) stores a series of multiple alignments at the DNA level between entire genomes. NET (.net) - the NET file format is used to describe the axtNet data that underlie the net alignment annotations in the UCSC Genome Browser. PDF (.pdf) - portable document format PNG (.png) - portable network graphics QMAP (.qmap) - QMAP files are generated for methylation data from an internal BGI pipeline. QUAL (.qual) - the QUAL file format represents base quality score file for NextGen data (similar in format to fasta). RPKM (.rpkm) - Gene expression levels are calculated by Reads Per Kilobase per Million (RPKM) mapped reads eg 1kb transcript with 1000 alignments in a sample of 10 million reads (out of which 8 million reads can be mapped) will have RPKM = 1000/(1 * 8) = 125: ENSP00000379387 15.5651433366423 6002951 289 3093 ENSP00000349977 24.7483107230444 6002951 398 2679 ENSP00000368887 24.6477413647837 6002951 174 1176 SAM (.sam) - the Sequence Alignment/Map (SAM) format is a TAB-delimited text format consisting of a header section, which is optional, and an alignment section. Most often it is generated as a human readable version of its sister BAM format, which stores the same data in a compressed, indexed, binary form. Currently, most SAM format data is output from aligners that read FASTQ files and assign the sequences to a position with respect to a known reference genome. In the future, SAM will also be used to archive unaligned sequence data generated directly from sequencing machines. See The Sequence Alignment/Map format and SAMtools TAR (.tar) - an archive containing other files TEXT (.doc, .readme, .text, .txt) - a text file VCF (.vcf) - the Variant Call Format (VCF) is a text file format for representing eg SNPs, InDels, CNVs, SVs, microsatellites, genotypes. WEGO (.wego) - the Web Gene Ontology (WEGO) Annotation format consists of the protein ID, followed by column(s) that are the GO ID(s): Bmb015379_2_IPR001092 Bmb003749_1_IPR006329 GO:0009168 GO:0003876 Bmb006173_1_IPR000909 GO:0007165 GO:0004629 GO:0007242< See WEGO: a web tool for plotting GO annotations WIG (.wig) - the output file from TopHat is a UCSC wigglegram of alignment coverage. UNKNOWN - any file format not in this list XML (.xml) - eXtensible Markup Language